internal/bundle/lint.go

```diff
@@ -387,31 +387,44 @@ func lintFile(file string) (*Results, error) {
 		for j := range policy.Groups {
 			group := policy.Groups[j]
 
-			// issue warning if no filters are assigned
-			if (group.Filters == nil || len(group.Filters.Items) == 0) &&
-				// but do not show the warning if the policy has variants or
-				// if the filters are set within the group
-				len(group.Policies) == 0 && !checksHaveFiltersOrVariants(group, globalQueriesByUid) {
-				location := Location{
-					File:   file,
-					Line:   group.FileContext.Line,
-					Column: group.FileContext.Column,
-				}
+			// issue warning if no filters are assigned but do not show the warning if
+			// the group has policies
+			if (group.Filters == nil || len(group.Filters.Items) == 0) && len(group.Policies) == 0 {
```

Comment:
I dont think the check on `len(group.Polilcies)` is needed here

Reply:
fixed!